LIB386/LIB_MIDI/MIDI.C

```diff
@@ -17,9 +17,9 @@
 #include <string.h>
```

Comment:
Same as above, I would create a preprocessor macro to prevent those to be used during compilation instead of commenting the code.